explorer/src/utils/index.ts

```diff
@@ -10,3 +13,11 @@ export default cn;
 export function isNotNullOrUndefined<T extends object>(input: null | undefined | T): input is T {
   return input !== null && input !== undefined;
 }
+
+export function getBlockExplorerLink(chain: Chain) {
+  if (chain.blockExplorers) {
+    return `${chain.blockExplorers.default.url}/address`;
+  } else {
+    return `${links[chain.id].address}/address`;
```

Comment:
All the links defined in `constants/index.ts` end with 'address' already, we don't need this one too